src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -853,6 +854,7 @@ void Transformations::PostLpt() {
 
     CPU_REGISTER_PASS_COMMON(postLPTPassManager, ov::pass::transpose_sinking::TSShapeOfForward);
     CPU_REGISTER_PASS_COMMON(postLPTPassManager, StatefulSDPAFusion);
+    CPU_REGISTER_PASS_X64(postLPTPassManager, ov::intel_cpu::SDPAFuseTransposeReshape);
```

Comment:
I don't see any x64 specific dependencies in the impl. as follow-up task can we try to make this optimization Common and see how it works on ARM platform?,

Reply:
hi @dmitry-gorokhov , I found Arm's SDPA kernel doesn't support memory permute.

If SDPA doesn't fuse concat, code will go into Arm implementation branch, we will get random result in each inference.